docs/social/online-matches/index.md

```diff
@@ -0,0 +1,266 @@
+---
+seo:
+  title: Matching Mutually Interested Online Dating Users
+  description: This recipe tracks repeated interactions between users of a social network or dating site.
+---
+
+# Matching Mutually Interested Online Dating Users
+
+_Dating sites are a very interesting use case. One of their core
+features is a messaging platform, and users chatting back and forth is an
+ideal fit for a real-time streaming system. But look at the site from
+another angle and it's all about making connections between people,
+so maybe a graph database is the right fit? Well, Kafka Connect makes
+it easy to use the strengths of both in tandem, but before we go that
+far, let's dig in and see how much graph-analysis we can do with some
+clever ksqlDB tricks._
+
+We'll create a messaging system, and then ask, "Which pairs of people
+have made proper connections?" Defining a 'proper' connection is a
+mystery for sociologists, but for our purposes we'll say two people
+have connected properly if A sends a message to B, B responds, and A
+follows-up. That should be enough chit-chat to get the ball rolling.
+
+# Solution
+
+``` sql
+--8<-- "docs/social/online-matches/process.sql"
+```
+
+## Explanation
+
+### Set Up Your Environment
+
+--8<-- "docs/shared/ccloud_setup.md"
+
+### Create The Initial Dataset
+
+``` sql
+--8<-- "docs/social/online-matches/manual.sql"
+```
+
+## Tracking Connections
+
+If you look at those `messages`, it's clear there are a lot of hellos
+bouncing around, but beyond that it's hard to see any patterns. Let's
+use some queries to make sense of it. We'll build up our answer to,
+"Who's connected to who?"  gradually. Before we begin, some session
+settings to make sure we all get the same results:
+
+```sql
+SET 'auto.offset.reset' = 'earliest';
+SET 'ksql.query.pull.table.scan.enabled' = 'true';
+```
+
+### Split By Conversation
+
+The first step is to break the stream up into individual
+conversations. If we sort the sender and receiver of each message, we
+can create a unique ID for every pair that chats (or tries to start
+chatting), and use that to group all the events:
+
+```sql
+CREATE TABLE conversations_v1 AS
```

Comment:
`_v1`, `_v2`, `_v3` is non-descriptive.  WDYT about renaming so they are more meaningful?  e.g. something along the lines of `pairings`, `last_sent`, etc?

Reply:
Yup, will do. 👍